packages/presentation/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
     sdk: flutter
   domain:
     path: ../domain
+  get_it: ^7.2.0
```

Comment:
no need in specifying the same dependency in all modules. u can specify it in the domain (for example) and all modules which contain the domain will have get_it. read about transitive dependencies